stacks/ambassador/deploy.sh

```diff
@@ -5,15 +5,15 @@ set -e
 ################################################################################
 # repo
 ################################################################################
-helm repo add datawire https://getambassador.io
-helm repo update > /dev/null
+helm repo add datawire https://app.getambassador.io
+helm repo update datawire > /dev/null
```

Comment:
should we specific a specific minimum Helm 3 version, or encourage to use latest 3? I tried to run this on my system using helm 3.6.x and it failed to install saying `helm repo update does take any parameters`. I upgraded to 3.8.x and it worked then

Reply:
I have removed `datawire` from the command `helm repo update > /dev/null` to be compatible with all helm 3 versions